### PR TITLE
[REF] cooperator : unbrand

### PR DESCRIPTION
**Pull request by robinkeunen**
https://github.com/coopiteasy/vertical-cooperative/issues/258

### CODE REVIEW

File: cooperator/migrations/12.0.5.0.0/pre-migration.py [hunk 1]
```diff
@@ -0,0 +1,19 @@
+import logging
+
+# for this major refactoring, we decided to depend on
+# openupgradelib
+from openupgradelib import openupgrade
```
Comment (robinkeunen) [inline cooperator/migrations/12.0.5.0.0/pre-migration.py:5]: @remytms @huguesdk @carmenbianca I'm a bit lost on what's the best way to list module dependencies 

- in vertical-cooperative/requirements.txt ?
- in modules `"external_dependencies"`

Comment (carmenbianca) [reply cooperator/migrations/12.0.5.0.0/pre-migration.py:5]: The second option, I am fairly certain. Or both.

Comment (huguesdk) [reply cooperator/migrations/12.0.5.0.0/pre-migration.py:5]: i think it’s both. we always install only from `requirements.txt` currently. i don’t know exactly how `external_dependencies` is used.

**Comment by robinkeunen** [comment]
@carmenbianca already merging the days work.

**Pull request merged**